src/main/java/com/auth0/jwk/Jwk.java

```diff
@@ -43,7 +43,7 @@
      * @param additionalAttributes additional attributes not part of the standard ones
      */
     @SuppressWarnings("WeakerAccess")
-    public Jwk(String id, String type, String algorithm, String usage, String operations, String certificateUrl, List<String> certificateChain, String certificateThumbprint, Map<String, Object> additionalAttributes) {
+    public Jwk(String id, String type, String algorithm, String usage, List<String> operations, String certificateUrl, List<String> certificateChain, String certificateThumbprint, Map<String, Object> additionalAttributes) {
```

Comment:
Although you're right and the ieft stands that the `key_ops` is an array, changing this line would result in a breaking change for existing users of the library. The existing constructor should call a new one with a `List` parameter instead, passing a `Collections.singletonList(operations)` for that value. The old one must be marked as deprecated and, ideally, javadoc should be added as well to explain this change.